src/components/victory-gauge.js

```diff
@@ -379,23 +499,33 @@ export default class VictoryGauge extends React.Component {
       props.colorScale : Style.getColorScale(props.colorScale);
     const padding = Helpers.getPadding(props);
     const radius = this.getRadius(props, padding);
-    const data = Helpers.getData(props);
-    const labelPosition = this.getLabelPosition(props, style, radius);
+    const segmentValues = this.getSegments(props);
+    const layoutFunction = this.getSliceFunction(props);
+    const segmentLocations = layoutFunction(segmentValues);
+    const gaugeRange = this.getGaugeRange(props, segmentLocations);
+
+    const tickValues = props.tickValues;
+    const tickCount = props.tickCount ? props.tickCount : tickValues.length;
     const pathFunction = d3Shape.arc()
       .outerRadius(radius)
       .innerRadius(props.innerRadius);
-    return {style, colors, padding, radius, data, labelPosition, pathFunction};
+    return {
+      style, colors, padding, radius,
+      tickCount, tickValues, pathFunction, segmentLocations, gaugeRange
+    };
 
   }
 
   render() {
     // If animating, return a `VictoryAnimation` element that will create
     // a new `VictoryBar` with nearly identical props, except (1) tweened
     // and (2) `animate` set to null so we don't recurse forever.
+
     if (this.props.animate) {
       const whitelist = [
-        "data", "endAngle", "height", "innerRadius", "padAngle", "padding",
-        "colorScale", "startAngle", "style", "width"
+        "data", "style", "startAngle", "endAngle", "colorScale",
+        "innerRadius", "outerRadius", "padAngle", "width", "height",
+        "padding", "tickValues", "tickFormat", "domain"
       ];
       return (
         <VictoryTransition animate={this.props.animate} animationWhitelist={whitelist}>
```

Comment:
You're going to want to use `VictoryAnimation` rather than `VictoryTransition` since this is single point data. `VictoryTransition` deals with changing numbers of data points